doc/design/paths-endpoint-webdav.md

```diff
@@ -0,0 +1,21 @@
+# Discussion on metadata needed for dandidav
+
+Submitted as a PR to avoid spagetti thread discussion on https://github.com/dandi/dandi-archive/issues/1837#issuecomment-1921864949
+
+
+@jwodder listed needed properties and metadata fields:
+
+* Asset properties used by dandidav:
+    * `asset_id`
+    * `blob_id`
+    * `zarr_id`
+    * `path`
+    * `size`
```

Comment:
```suggestion
    * `size`  (PRESENT)
```

Reply:
`size`, `created`, `modified`, `blob_id`, `zarr_id`, and metadata are not present in `.../assets/paths/` responses.  `dandidav` currently gets those from just `.../assets/`.

Reply:
`aggregate_size`?

Reply:
`dandidav` currently ignores that and just gets the size from the properties.

Reply:
ok, but my point is that the value is available already albeit under obfuscated name somewhat.